Gemfile

```diff
@@ -1,4 +1,7 @@
 source "https://rubygems.org"
 
 # Specify your gem's dependencies in carrierwave-base64.gemspec
+
+# multiple file uploader support
+gem 'carrierwave', :github => 'carrierwaveuploader/carrierwave'
```

Comment:
Do you consider to use `:git` directive since `:github` directive can be compromised by man-in-the-middle attack. in bundler 1.x

https://sikac.hu/secure-your-github-sources-in-your-gemfile-f99784015200#.ccouxtaxc